docs/user/filter_expressions.rst

```diff
@@ -0,0 +1,205 @@
+   limited to:
+
+   * ``'release'``
+   * ``'aurora'``
+   * ``'beta'``
+   * ``'nightly'``
+   * ``'default'`` (self-built or automated testing builds)
+
+.. js:attribute:: normandy.isDefaultBrowser
+
+   Boolean specifying whether Firefox is set as the user's default browser.
+
+.. js:attribute:: normandy.searchEngine
+
+   **Example:** ``'google'``
+
+   String containing the user's default search engine identifier.
+
+.. js:attribute:: normandy.syncSetup
+
+   Boolean containing whether the user has set up Firefox Sync.
+
+.. js:attribute:: normandy.plugins
+
+   An object mapping of plugin names to :js:class:`Plugin` objects describing
+   the plugins installed on the client.
+
+.. js:attribute:: normandy.locale
+
+   **Example:** ``'en-US'``
+
+   String containing the user's locale.
+
+.. js:attribute:: normandy.country
+
+   **Example:** ``'US'``
+
+   `ISO 3166-1 alpha-2`_ country code for the country that the user is located
+   in. This is determined via IP-based geolocation.
+
+   .. _ISO 3166-1 alpha-2: https://en.wikipedia.org/wiki/ISO_3166-1
+
+.. js:attribute:: normandy.request_time
+
+   Date object set to the time and date that the user requested recipes from
+   Normandy. Useful for comparing against date ranges that a recipe is valid
+   for.
+
+   .. code-block:: javascript
+
+      // Do not run recipe after January 1st.
+      normandy.request_time < '2011-01-01'|date
+
+Transforms
+----------
+This section describes the transforms available to filter expressions, and what
+they do. They're documented as functions, and the first parameter to each
+function is the value being transformed.
+
+.. js:function:: date(dateString)
+
+   Parses a string as a date and returns a Date object. Strings should be in a
+   format recognized by `Date.parse()`_.
+
+   :param string dateString:
+      String to parse as a date.
+
+   .. code-block:: javascript
+
+      '2011-10-10T14:48:00'|date // == Date object matching the given date
+
+   .. _Date.parse(): https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse
+
+Examples
+--------
+This section lists some examples of commonly-used filter expressions.
+
+.. code-block:: javascript
+
+   // Match users using the en-US locale while located in India
+   normandy.locale == 'en-US' && normandy.country == 'IN'
+
+   // Match users in any English locale using Firefox Beta
+   (
+      normandy.locale in ['en-US', 'en-AU', 'en-CA', 'en-GB', 'en-NZ', 'en-ZA']
+      && normandy.channel == 'beta'
+   )
+
+   // Only run the recipe between January 1st, 2011 and January 7th, 2011
+   (
+      normandy.request_time > '2011-01-01T00:00:00+00:00'|date
+      && normandy.request_time < '2011-01-07T00:00:00+00:00'|date
+   )
+
+   // Match users located in the US who have Firefox as their default browser
+   normandy.country == 'US' && normandy.isDefaultBrowser
+
+   // Match users with the Flash plugin installed
+   normandy.plugins['Shockwave Flash']
```

Comment:
This might need some explaining for people who aren't familiar with JS's truthiness behaviors.